storage/integration_test.go

```diff
@@ -3023,7 +3025,8 @@ func TestIntegration_PublicBucket(t *testing.T) {
 	}
 
 	errCode := func(err error) int {
-		err2, ok := err.(*googleapi.Error)
+		var err2 *googleapi.Error
+		ok := xerrors.As(err, &err2)
 		if !ok {
 			return -1
 		}
```

Comment:
nit: Can this be inlined as follows?
```suggestion
		var err2 *googleapi.Error
		if ok := xerrors.As(err, &err2); !ok {
			return -1
		}
```

Also I don't love the variable name `err2` but I realize that was there already.

Reply:
Good idea, fixed.